### ISSUE TITLE
请问什么是emb mode呢？

### ISSUE THREAD

**Issue by kendo6666** [ashawkey/RAD-NeRF#31]
在main.py中，有一个emb模式的参数，请问它的具体功能是什么呢？在什么样的场景下使用呢？谢谢指点

**Comment by ashawkey** [comment]
@kendo6666 Hi, it's not meant to be used. Basically it means we provide audio features as class labels and learn an embedding per class instead of directly passing in per class logits. As far as I have tested, it generates worse results.
